synfig-core/src/synfig/valuenodes/valuenode_range.h

```diff
@@ -56,9 +56,13 @@ class ValueNode_Range : public LinkableValueNode
 	virtual String get_name()const;
 	virtual String get_local_name()const;
 
+	//! Checks if it is possible to call get_inverse() for target_value at time t.
+	//! If so, return the link_index related to the return value provided by get_inverse()
+	virtual InvertibleStatus is_invertible(Time t, const ValueBase& target_value, int* link_index = nullptr) const;
 	//! Returns the modified Link to match the target value at time t
-	ValueBase get_inverse(Time t, const synfig::Vector &target_value) const;
-	ValueBase get_inverse(Time t, const synfig::Angle &target_value) const;
+	virtual ValueBase get_inverse(Time t, const synfig::ValueBase &target_value) const;
+	virtual ValueBase get_inverse(Time t, const synfig::Vector &target_value) const;
```

Comment:
Does `vector` and `angle` methods is still required here?

Reply:
I kept them because they are 'long' methods. Maybe change them to private access?

Reply:
> Maybe change them to private access?

Agree

Reply:
Done